draft-ietf-tls-tls13.md

```diff
@@ -709,12 +709,12 @@ use the established keys to protect application layer traffic.
 
 TLS supports three basic key exchange modes:
 
-- Diffie-Hellman (of both the finite field and elliptic curve
+- Diffie-Hellman (both the finite field and elliptic curve
   varieties).
 
-- A pre-shared symmetric key (PSK)
+- A pre-shared symmetric key (PSK).
 
-- A combination of a symmetric key and Diffie-Hellman
+- A combination of a symmetric key and Diffie-Hellman.
```

Comment:
nit of a nit: none of these bullet points are sentences, thus none really should have periods. I'd suggest not adding the latter two and dropping the first instead.